Makefile

```diff
@@ -60,7 +60,7 @@ populate-deployment-variables:
 	eval "$$(make aws-assume-role-export-variables)"
 	echo "export DB_SERVER=$$(make -s aws-rds-describe-instance-value DB_INSTANCE=$(DB_SERVER_NAME) KEY_DOT_PATH=Endpoint.Address)"
 	echo "export DB_USER_NAME=$$(make -s secret-get-existing-value NAME=$(DB_USER_NAME_SECRET_NAME) KEY=$(DB_USER_NAME_SECRET_KEY))"
-	echo "export URL_SLACK_WEBHOOK=$$(make -s secret-get-existing-value NAME=$(SLACK_WEBHOOK_SECRET_NAME) KEY=$(SLACK_WEBHOOK_SECRET_KEY))"
+	echo "export SLACK_WEBHOOK_URL=$$(make -s secret-get-existing-value NAME=$(SLACK_WEBHOOK_SECRET_NAME) KEY=$(SLACK_WEBHOOK_SECRET_KEY))"
```

Comment:
there was a reason this started with URL, something to do with variables exported to docker, possibly line 367 of docker.mk

Reply:
This has been resolved in the serverless.mk file